_devs/BrendaHoyle.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Brenda Hoyle
+github: BrendaHoyle
+urls:
+    -   name: Center Centre cohort 1 blog
```

Comment:
this name actually needs to say ```website``` This lets the jeykll app know what icon to put for the url you provided. The name doesn't actually show up in the ui.